main.go

```diff
@@ -65,12 +66,19 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	watchNamepace := os.Getenv("WATCH_NAMESPACE")
+	if watchNamepace == "" {
+		setupLog.Error(fmt.Errorf("WATCH_NAMESPACE env variable must be set"), "unable to get WATCH_NAMESPACE")
```

Comment:
I'd rather pass nil as the first field, than needing to craft an error for logging purpouses only

Reply:
done